extensions/vertx/deployment/src/test/java/io/quarkus/vertx/DuplicatedContextTest.java

```diff
@@ -147,12 +148,14 @@ public void testThatBlockingEventConsumersAreCalledOnDuplicatedContext() {
         @ConsumeEvent(value = "context")
         Uni<String> receive(String data) {
             Assertions.assertTrue(Thread.currentThread().getName().contains("vert.x-eventloop"));
+            VertxContextSafetyToggle.validateContextIfExists("Not marked as safe", "Not marked as safe");
```

Comment:
Hm, would it make sense to add a `VertxContextSafetyToggle.validateContextIfExists()` method which only accepts one string used for both the `errorMessageOnVeto` and `errorMessageOnDoubt`? It would save some typing ;-)

Reply:
Here it's just for a test. 
When you want to really check, ou need two different messages.

Reply:
> Here it's just for a test.

Yep, I know... but it would help with tests at least ;-)